student.rb

```diff
@@ -1,9 +1,9 @@
 require './person'
 
-class Studentes < Person
+class Student < Person
   attr_reader :classroom
 
-  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+  def initialize(classroom, age, name: 'unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
   end
```

Comment:
Great job @PedroH-Neves 🤝

- [ ] You have done well. However, the app breaks whenever I try to add a classroom. Kindly fix this and resubmit your project for approval.
![classroom](https://github.com/PedroH-Neves/OOP-school/assets/63917725/a87a04ac-b958-4232-a9ef-95733f4cfe65)